src/providers/Environment.ts

```diff
@@ -1,24 +1,54 @@
 import * as path from 'path';
 import * as dotenv from 'dotenv';
+import validator from 'validator';
 
 import Logger from '../libs/Logger';
 
-const requiredENV = ['NODE_ENV', 'DATABASE_URL', 'DISCORD_TOKEN', 'PRIVATE_BOT', 'OSU_API_KEY'];
```

Comment:
I think the osu! module is going to crash if the API key isn't defined. Make change to the osu! module to handle this too

Reply:
DEVELOPER_IDS too, but I already pushed a change that will handle this

Reply:
Done~!